### ISSUE TITLE
[steamshovel] partial closing of open files is broken (Trac #1534)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1526]
steamshovel behaves very strangely if one uses the 'close only file that was opened last' feature and opens another file. it shows frames from the file that was already closed instead of new frames from the new file and also the number of frames in the file can be off.

Migrated from https://code.icecube.wisc.edu/ticket/1534
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:15", 
    "description": "steamshovel behaves very strangely if one uses the 'close only file that was opened last' feature and opens another file. it shows frames from the file that was already closed instead of new frames from the new file and also the number of frames in the file can be off.", 
    "reporter": "hdembinski", 
    "cc": "david.schultz", 
    "resolution": "fixed", 
    "_ts": "1458335655846260", 
    "component": "combo core", 
    "summary": "[steamshovel] partial closing of open files is broken", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-01-28T22:18:41", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160129T02:02:31: 
* hdembinski changed status from "new" to "closed"
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 141325:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141325"
> critical bug-fix, closes #1526
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:15: 
* olivas commented:

> Milestone February 2016 deleted


* olivas changed milestone from "February 2016" to ""
